### DISCUSSION TITLE
Does langfuse uses any Postgres-only features?

### DISCUSSION THREAD

**Discussion by rohts-patil** [langfuse/langfuse#2069] [Support] [answered]
Does langfuse use any Postgres-only features? I have MariaDB with me and wanted to check if I can use mariaDB instead of Postgres for self-hosting purposes.

**Comment by marcklingen** [comment, marked as answer]
yes, many unfortunately. supporting multiple db engines is out of scope for now as we prepare for langfuse v3: https://github.com/orgs/langfuse/discussions/1902
